lua/sql/defs.lua

```diff
@@ -155,416 +146,28 @@ M.flags['open_privatecache']  = 0x00040000
-M.flags['fcntl_lock_timeout']          = 34
-M.flags['fcntl_data_version']          = 35
-M.flags['fcntl_size_limit']            = 36
-M.flags['fcntl_ckpt_done']             = 37
-M.flags['fcntl_reserve_bytes']         = 38
-M.flags['fcntl_ckpt_start']            = 39
-
--- Flags for the xAccess VFS method
-M.flags['access_exists']    = 0
-M.flags['access_readwrite'] = 1
-M.flags['access_read']      = 2
-
--- Flags for the xShmLick VFS method
-M.flags['shm_unlock']    = 1
-M.flags['shm_lock']      = 2
-M.flags['shm_shared']    = 4
-M.flags['shm_exclusive'] = 8
-
--- Maximum xShmLock index
-M.flags['shm_nlock'] = 8
-
--- Configuration Options
-M.flags['config_singlethread']        = 1
-M.flags['config_multithread']         = 2
-M.flags['config_serialized']          = 3
-M.flags['config_malloc']              = 4
-M.flags['config_getmalloc']           = 5
-M.flags['config_scratch']             = 6
-M.flags['config_pagecache']           = 7
-M.flags['config_heap']                = 8
-M.flags['config_memstatus']           = 9
-M.flags['config_mutex']               = 10
-M.flags['config_getmutex']            = 11
-M.flags['config_lookaside']           = 13
-M.flags['config_pcache']              = 14
-M.flags['config_getpcache']           = 15
-M.flags['config_log']                 = 16
-M.flags['config_uri']                 = 17
-M.flags['config_pcache2']             = 18
-M.flags['config_getpcache2']          = 19
-M.flags['config_covering_index_scan'] = 20
-M.flags['config_sqllog']              = 21
-M.flags['config_mmap_size']           = 22
-M.flags['config_win32_heapsize']      = 23
-M.flags['config_pcache_hdrsz']        = 24
-M.flags['config_pmasz']               = 25
-M.flags['config_stmtjrnl_spill']      = 26
-M.flags['config_small_malloc']        = 27
-M.flags['config_sorterref_size']      = 28
-M.flags['config_memdb_maxsize']       = 29
-
--- Database Connection Configuration Options
-M.flags['dbconfig_maindbname']            = 1000
-M.flags['dbconfig_lookaside']             = 1001
-M.flags['dbconfig_enable_fkey']           = 1002
-M.flags['dbconfig_enable_trigger']        = 1003
-M.flags['dbconfig_enable_fts3_tokenizer'] = 1004
-M.flags['dbconfig_enable_load_extension'] = 1005
-M.flags['dbconfig_no_ckpt_on_close']      = 1006
-M.flags['dbconfig_enable_qpsg']           = 1007
-M.flags['dbconfig_trigger_eqp']           = 1008
-M.flags['dbconfig_reset_database']        = 1009
-M.flags['dbconfig_defensive']             = 1010
-M.flags['dbconfig_writable_schema']       = 1011
-M.flags['dbconfig_legacy_alter_table']    = 1012
-M.flags['dbconfig_dqs_dml']               = 1013
-M.flags['dbconfig_dqs_ddl']               = 1014
-M.flags['dbconfig_enable_view']           = 1015
-M.flags['dbconfig_legacy_file_format']    = 1016
-M.flags['dbconfig_trusted_schema']        = 1017
-M.flags['dbconfig_max']                   = 1017
-
--- Authorizer Return Codes
-M.flags['deny']   = 1
-M.flags['ignore'] = 2
-
--- Authorizer Action Codes
```

Comment:
Hmm just wondering what are these even for and when they do get return? is it important for our sql wrappers?

Reply:
I think for `sqlite3_user_authenticate` or similar

Reply:
No when doing int `sqlite3_set_authorizer`

Reply:
https://sqlite.org/c3ref/c_alter_table.html

Reply:
Oh, nice. I'm excited for a day we need suck a thing or a user do feature request :D

Reply:
We can always add stuff back 😆 But that are just the result codes so the function should still be callable.